CHANGES.txt

```diff
@@ -1,4 +1,6 @@
 3.0.24:
+ * Prevent unbounded number of pending flushing tasks (CASSANDRA-16261)
+ * Allow empty string in collections with COPY FROM in cqlsh (CASSANDRA-16372)
```

Comment:
@ekaterinadimitrova2 Are the bits from CASSANDRA-16372 and CASSANDRA-15563 supposed to be here?

CC @adelapena

Reply:
Maybe this is caused by the branch being (re)based on `apache/cassandra-3.0`, while this PR is against `ekaterinadimitrova2/cassandra-3.0`, which is slightly outdated?

Reply:
Yeah, I rebased many times these but I think this project is too active :D Every time there are a lot of conflicts.... I will rebase and push again CI and see where we are. Thank you!